data.sql

```diff
@@ -55,4 +55,30 @@ select * from animals where escape_attempts =0;
 select avg(weight_in_kg) from animals;
 select name min(weight_in_kg) and max(weight_in_kg) from animals group by name;
 select neutered, max(escape_attempts) from animals group by neutered;
-select name, avg(escape_attempts) from animals where date_of_birth BETWEEN '1990-1-1' AND '2000-12-31' group by name;
\ No newline at end of file
+select name, avg(escape_attempts) from animals where date_of_birth BETWEEN '1990-1-1' AND '2000-12-31' group by name;
+
```

Comment:
Please be sure to add the following data to your vets table as it is part of the requirements 🤓

![image](https://user-images.githubusercontent.com/15807343/190691917-430d3517-1edb-4a55-8638-690851bb4d35.png)

Reply:
@mateo951 Thanks for the review, I populated my DB  with vets data but forgot to type out my queries in vscode